hail/src/main/scala/is/hail/rvd/RVDPartitioner.scala

```diff
@@ -10,6 +11,7 @@ import org.apache.spark.sql.Row
 import org.apache.spark.{Partitioner, SparkContext}
 
 class RVDPartitioner(
+  val sm: HailStateManager,
```

Comment:
Got a friendly build failure from this :) Can you explain why partitioners need to know about reference genomes?